definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -1983,6 +1983,24 @@
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/update-task-list/submitted",
     "EndButtonLabel": "Save and continue",
     "EventEnablingCondition": "caseTypeOfApplication=\"C100\" AND taskListVersion=\"v3\""
+  },
+  {
+    "LiveFrom": "26/07/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "removeDraftOrder",
+    "Name": "Remove draft order",
+    "Description": "Remove draft order",
+    "DisplayOrder": 32,
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-remove-draft-order-dropdown",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/remove-draft-order/about-to-submit",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/remove-draft-order/submitted",
```

Comment:
We will need only one callback here . Please manage to have an aboutTosubmit only .

Reply:
Hi @pragnesh-patel3 did we manage to keep just 1 callback as per review comments from @priteshkr . Checking this since I can still see 2 of them.
If yes can you please resolve the conversation..